### PR TITLE
MS-1463 check yours answers so far basic

### PR DESCRIPTION
**Pull request by jjrobothamho**
- Update Read report behaviour
- add CYA page content

### CODE REVIEW

File: apps/nrm/views/check-your-answers-so-far.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#hasReports}}
+      {{> partials-check-your-answers-so-far-summary-table}}
+      {{#input-submit}}continue{{/input-submit}}
+    {{/hasReports}}
+    {{^hasReports}}
+    Error
```
Comment (sulthan-ahmed) [inline apps/nrm/views/check-your-answers-so-far.html]: what scenario would this appear?

File: apps/nrm/views/partials/check-your-answers-so-far-summary-table-potential-victim.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+          <td></td>
+        </tr>
+      {{/values.does-pv-need-support}}
+      {{/isDutyToNotifyPath}}
+
+      {{#values.pv-want-to-submit-nrm}}
+        <tr>
+          <td>{{#t}}pages.confirm.fields.pv-want-to-submit-nrm.label{{/t}}</td>
+          <td class="no-white-space">
+            <span>{{formattedPvWnatToSubmitNrm}}</span>
```
Comment (sulthan-ahmed) [inline apps/nrm/views/partials/check-your-answers-so-far-summary-table-potential-victim.html]: Wnat?

Comment (jjrobothamho) [reply apps/nrm/views/partials/check-your-answers-so-far-summary-table-potential-victim.html]: lol cheers think I need to edit that one throughout @cmedlandho  could create a JIRA issue for this when you have some time. Think we will need to edit all occurrences of that.

File: apps/nrm/views/check-your-answers-so-far.html (hunk 1 above)
Comment (jjrobothamho) [reply apps/nrm/views/check-your-answers-so-far.html]: If for whatever reason the session gets destroyed or the user goes directly to the url without going through the app?

Is this a bit too much?

**Comment by jjrobothamho** [comment]
I'll merge for now and tackle those issues in subsequent PR's

File: apps/nrm/views/check-your-answers-so-far.html (hunk 1 above)
Comment (sulthan-ahmed) [reply apps/nrm/views/check-your-answers-so-far.html]: I think it's fine, we will just test it

File: apps/nrm/views/partials/check-your-answers-so-far-summary-table-potential-victim.html (hunk 2 above)
Comment (sulthan-ahmed) [reply apps/nrm/views/partials/check-your-answers-so-far-summary-table-potential-victim.html]: ahh no worries

**Comment by sulthan-ahmed** [comment]
makes sense to tackle later

**Pull request merged**